docker-hub/api/dvp.yaml

```diff
@@ -0,0 +1,562 @@
+openapi: 3.0.0
+info:
+  title: DVP Data API
+  version: 1.0.0
+  x-logo:
+    url: https://docs.docker.com/images/logo-docker-main.png
+    href: /reference
+  description: |
+    The Docker DVP Data API allows Docker Verified Publishers to view image pull analytics data for their namespaces
+servers:
+  - url: https://hub-stage.docker.com/api/dvp/v1
+security:
+  - HubAuth: []
+
+tags:
+  - name: namespaces
+    x-displayName: Namespace data
+  - name: repos
+    x-displayName: Repository data
+  - name: discovery
+    x-displayName: Discovery
+
+x-tagGroups:
+  - name: API
+    tags:
+      - discovery
+      - namespaces
+      - repos
+
+paths:
+  /data:
+    get:
+      tags: [discovery]
+      summary: Get namespaces and repos
+      description: Gets a list of namespaces and repos which have data available
+      operationId: getNamespaces
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NamespaceData'
+  /data/{namespace}:
+    get:
+      tags: [discovery]
+      summary: Get namespace
+      description: Gets metadata associated with specified namespace
```

Comment:
Can we explain in plain english what this 'metadata' is and why a user wants it? Is it just the extra repos they have access to right now, with the option to extend further in future? Is it clear what the flow of using this data for subsequent calls is?